app/src/main/java/com/theopensourcefamily/chessclock/ClocksActivity.kt

```diff
@@ -36,9 +38,10 @@ class ClocksActivity : AppCompatActivity(), ClocksView {
 
   override val userInteractions: Observable<ClocksView.Interaction>
     get() = Observable.merge(
-      blackClock.clicks().map { ClocksView.Interaction.BlackPressed },
-      whiteClock.clicks().map { ClocksView.Interaction.WhitePressed },
-      pauseButton.clicks().map { ClocksView.Interaction.StopPressed }
+      blackClock.touches().map { ClocksView.Interaction.BlackPressed },
```

Comment:
What is the difference between touches and clicks?

Reply:
The difference is that touches will react on MotionEvents (action_up, action_down. action_move, etc), and clicks will react on click.

And the difference beetween click events and touch events is that click is a set of motion events, and touch is a single one, for example if you just click it will add two events to "touches" (action down and action up), and one click, also if you touch, move the finger over the widget, touches keep being detected, until action_up is performed and then onClick is called.

Sumarizing: 
- ```MotionEvent.ACTION_DOWN``` *(```onTouch``` is called here)*
- List of events != ```MotionEvent.ACTION_UP``` *(```onTouch``` is called here for every event)*
- ```MotionEvent.ACTION_UP```. *(```onClick``` is called here)*

Reply:
@all-contributorsbot please add @helbaroudy for review

Reply:
@all-contributors please add @helbaroudy for review

Reply:
@all-contributors please add @helbaroudy for review

Reply:
Thank you for your response! :) 
Everything clear now